README.md

```diff
@@ -100,6 +100,16 @@ $ sudo apt-get install lttng-modules-dkms
 
 For more information about LTTng, [refer to its documentation](https://lttng.org/docs/v2.13/).
 
+### Disabling the instrumentation
+
+To disable all instrumentation in runtime, set the `TRACETOOLS_RUNTIME_DISABLE` envvar:
```

Comment:
Also, I feel like "disable all instrumentation at runtime" isn't really what this mechanism does, and it might be confusing because of the typical LTTng (runtime) trace events configuration mechanism.

What about something closer to what you're describing in the PR description (and what you're actually trying to achieve)? Something like:

```suggestion
To avoid loading the tracer at runtime (and therefore disable all instrumentation), set the `TRACETOOLS_RUNTIME_DISABLE` environment variable:
```